Shared/ContentView.swift

```diff
@@ -35,23 +35,31 @@ struct ContentView: View {
                         .focused($amountIsFocused)
                     
                     Picker("Number of people", selection: $numberOfPeople) {
-                        ForEach(2 ..< 100) {
+                        ForEach(2 ..< 101) {
```

Comment:
Maybe we should remove these magic numbers inside the ForEach

Reply:
Resolved